pytorch_lightning/accelerators/cpu_accelerator.py

```diff
@@ -47,6 +47,8 @@ def setup(self, model):
         # allow for lr schedulers as well
         self.setup_optimizers(model)
 
+        self.trainer.convert_to_lightning_optimizers()
```

Comment:
it means all, right?
```suggestion
        self.trainer.convert_to_lightning_optimizers()
```
```suggestion
        self.trainer.convert_all_to_lightning_optimizers()
```

Reply:
I think it is implicit it is all